docs/partial_source/deep_dive/ivy_frontends.rst

```diff
@@ -199,18 +205,51 @@ Ivy does not support the unique naming of individual operations, and so we omit
 .. code-block:: python
 
     # in ivy/functional/frontends/tensorflow/math.py
+    @to_ivy_arrays_and_back
     def tan(x, name=None):
         return ivy.tan(x)
 
 Likewise, :code:`tan` is also placed under :mod:`math`.
 By referring to the `tf.math.tan`_ documentation, we add the same arguments, and simply wrap :func:`ivy.tan` in this case.
 Again, we do not support the :code:`name` argument for the reasons outlined above.
 
+**NOTE**
+
+For The :mod:`tf.raw_ops` many functions posses the same behavior as those of :mod:`tf` namespace except that only the functions from :mod:`tf.raw_ops` takes key-word only arguments.
+For this we have a designed decorator to wrap the :mod:`tf` namespace function to satisfy this behavior. Let's take an example of :func:`tf.math.argmax` and :func:`tf.raw_ops.ArgMax`.
+
+.. code-block:: python
+    
+    # in ivy/functional/frontends/tensorflow/math.py
+    @to_ivy_arrays_and_back
+    def argmax(input, axis, output_type=None, name=None):
+        if output_type in ["uint16", "int16", "int32", "int64"]:
+            return ivy.astype(ivy.argmax(input, axis=axis), output_type)
+        else:
+            return ivy.astype(ivy.argmax(input, axis=axis), "int64")
+
+This function is a frontend ivy implementation of :func:`tf.math.argmax`, the similar function in :mod:`tf.raw_ops` has similar inputs and outputs except the :func:`tf.raw_ops.ArgMax` takes all inputs as key-word only.
+Let's take a look on how we can handle such case.
+
+.. code-block:: python
+
+    # in ivy/functional/frontends/tensorflow/raw_ops.py
+    ArgMax = to_ivy_arrays_and_back(
+        map_raw_ops_alias(
+            tf_frontend.math.argmax,
+            kwargs_to_update={"dimension": "axis"},
```

Comment:
I would explain that this is a dictionary, and how to use it in the general sense